tce/src/base/osal/OperationPoolPimpl.cc

```diff
@@ -147,6 +147,9 @@ OperationPoolPimpl::operation(const char* name) {
 
     Operation* effective = index_->effectiveOperation(name);
     if (effective != NULL) {
+        if (StringTools::stringToLower(name) == "hwloop") {
+            effective->setControlFlowOperation(true);
```

Comment:
Why monkey patch it here? Don't we have an .opp property for it?

Reply:
fixed!